### PR TITLE
[NFV] Add Hugepages configuration

### PR DESCRIPTION
**Pull request by jlausuch**
- Verification run: 
http://loewe.arch.suse.de/tests/1098#

### CODE REVIEW

File: tests/kernel/mellanox_ofed.pm [hunk 1]
```diff
@@ -31,6 +31,8 @@ sub run {
         zypper_call('--quiet in openvswitch python-devel python2-libxml2-python libopenssl1_1 insserv-compat libstdc++6-devel-gcc7 createrepo_c rpm', timeout => 500);
     }
     elsif (check_var('VERSION', '12-SP4')) {
```
Comment (asmorodskyi) [inline tests/kernel/mellanox_ofed.pm]: I thought we already clarify that we adding SDK during installation, not manually

File: tests/nfv/hugepages_config.pm [hunk 2]
```diff
@@ -0,0 +1,36 @@
...
+sub run {
+    select_console 'root-ssh';
+
+    my $hugepages  = get_required_var('HUGEPAGES');
+    my $hugepagesz = get_required_var('HUGEPAGESZ');
+    my $grub_file  = '/boot/grub2/grub.cfg';
+    my $boot_line  = "default_hugepagesz=$hugepagesz hugepagesz=$hugepagesz hugepages=$hugepages";
+
+    record_info("Kernel boot command");
+    assert_script_run("sed -i 's/showopts/showopts " . $boot_line . "/' " . $grub_file);
```
Comment (asmorodskyi) [inline tests/nfv/hugepages_config.pm]: > > Probably we also need to run something like `make grub`, right?

File: tests/kernel/mellanox_ofed.pm (hunk 1 above)
Comment (jlausuch) [reply tests/kernel/mellanox_ofed.pm]: Adding the variable WORKAROUND_MODULES=base,sdk didn't make the magic.. http://loewe.arch.suse.de/tests/1068#step/mellanox_ofed/8

Maybe we are missing some other parameter or some config, I don't know, but I add it back until we know a better solution.

File: tests/nfv/hugepages_config.pm (hunk 2 above)
Comment (jlausuch) [reply tests/nfv/hugepages_config.pm]: according to some instructions I found to set this up, a simple reboot is enough.. but that's why I still need to test it and it's WIP :)

Comment (jlausuch) [reply tests/nfv/hugepages_config.pm]: grub2-mkconfig was missing

**Pull request merged**